pmm/python.cmake

```diff
@@ -6,6 +6,7 @@ function(_pmm_find_python3 ovar)
         file(GLOB pyenv_dirs "$ENV{HOME}/.pyenv/versions/3.*/")
     endif()
     file(GLOB c_python_dirs "C:/Python3*")
+    list(REVERSE c_python_dirs)# Use the latest version instead of the oldest version
```

Comment:
I don't think CMake's `file(GLOB ...)` makes any guarantee about file order. Perhaps it would be better to use `list(SORT ...)` or walk through the list to find the maximum version (idk how python defines its versions, so just sorting and reversing may not be sufficient)

Reply:
I've read somewhere that on windows `file(GLOB ...)` always returns a sorted list, so I think it is unnecessary to resort the list again

Reply:
["The results will be ordered lexicographically"](https://cmake.org/cmake/help/latest/command/file.html#glob). Indeed, you are correct.

It will fail once Python reaches version 3.10, because then the lexicographical sort will place Python310 after Python39, but that's far off, so let's deal with that when we get to it.